src/pages/Products.js

```diff
@@ -1,238 +1,198 @@
-					setFilter={setSelectedFilter}
-					currentFilter={selectedFilter}
-				/>
-				<div className="w-full lg:w-4/5 space-y-8">
-					{selectedFilter === "none" && (
-						<p>
-							<b>Pro Tip: </b>Usa un filtro a la izquierda para limitar la
-							búsqueda
-						</p>
-					)}
-					{selectedFilter === "generic" && (
-						<div style={{backgroundColor: "#11698E"}} className="bg-add text-white md:mr-8 lg:mr-20 xl:mr-3 2xl:mr-10">
-							<h2 className="text-4xl px-4 py-2 font-bold">Genéricos</h2>
-						</div>
-					)}
-					{selectedFilter === "category" && (
-						<>
-							<div className="pl-8 lg:pl-2">
-								<div className="container items-center">
-									<div className="w-full">
-										<p className="text-text text-xl sm:text-2xl font-normal">
-										<Link to="/"> Inicio / </Link>
-										<Link to="/products"> Productos / </Link>
-										<Link to={`products?category=${selectedCategory.id}` }> {selectedCategory?.name} </Link>
-											 /{" "}
-											{selectedSubcategory?.name}
-										</p>
-									</div>
-								</div>
-							</div>
-							<div style={{backgroundColor: "#11698E"}} className="bg-add text-white md:mr-8 lg:mr-20 xl:mr-3 2xl:mr-10">
-								<h2 className="text-4xl px-4 py-2 font-bold">
-									{selectedCategory.name}{" "}
-									{!!selectedSubcategory?.name &&
-										`- ${selectedSubcategory.name}`}
-								</h2>
-							</div>
-							{!!selectedCategory.subcategories.length && (
-								<Slider
-									categories={categories}
-									data={selectedCategory.subcategories}
-									itemClick={subcategoryClicked}
-								/>
-							)}
-						</>
-					)}
-					{selectedFilter === "manufacturer" && (
-						<div style={{backgroundColor: "#11698E"}} className="bg-add text-white md:mr-8 lg:mr-20 xl:mr-3 2xl:mr-10">
-							<h2 className="text-4xl max-w-full block overflow-hidden text-ellipsis px-4 py-2 font-bold">
-								Fabricantes
-							</h2>
-						</div>
-					)}
-					{
-						currentProducts.length === 0
-						? <h1 className="text-center">No hay productos usando estos argumentos</h1>
-						:
-						<>
-							<div className="grid md:grid-cols-2 xl:grid-cols-4 gap-4 w-full p-0 justify-evenly xl:justify-around">
-								{
-									currentPosts.map((product) => (
-										<ProductItem data={product} key={`product-item-${product.id}`} />
-									))
-								}
-							</div>
-
-							{currentProducts.length <= 10
-								||
-								<Pagination
-									postsPerPage={postsPerPage}
-									totalPosts={currentProducts.length}
-									paginate={paginate}
-									currentPage={currentPage}
-									paginateFront={paginateFront}
-									paginateBack={paginateBack}
-								/>
-							}
-						</>
-					}
-				</div>
-			</div>
-			<Footer />
-		</>
-	);
+  const { search } = useLocation();
+  const { push } = useHistory();
+  const [products, setProducts] = useState([]);
+  const [currentProducts, setCurrentProducts] = useState([]);
```

Comment:
I think we should change the name of the file and functions for "artwork" / "artworks"